packages/contracts/test/dollar/core/CreditNFTManager.t.sol

```diff
@@ -209,7 +205,7 @@ contract CreditNFTManagerTest is LocalTestHelper {
             mockMessageSender, 2e18, expiryBlockNumber
         );
         UbiquityDollarManager(dollarManagerAddress).grantRole(
-            keccak256("GOVERNANCE_TOKEN_MINTER_ROLE"), creditNFTManagerAddress
+            keccak256("UBQ_MINTER_ROLE"), creditNFTManagerAddress
```

Comment:
Refer to updated role names comments above.

Reply:
Fixed